### PR TITLE
Lidarr: flac2mp3 initial release

### PR DESCRIPTION
**Pull request by TheCaptain989**
Add ffmpeg and script to convert audio to MP3s

### CODE REVIEW

**Comment by aptalca** [comment]
I tried to rebase this on a new branch based on the template branch but it requires cli resolution of the conflicts

**Comment by TheCaptain989** [comment]
@aptalca, okay.  I rebased this to the new template branch and fixed the conflicts.

File: .github/workflows/BuildImage.yml [hunk 1]
```diff
@@ -6,8 +6,13 @@ jobs:
   build:
     env:
       DOCKERHUB: "linuxserver/mods" #don't modify
+<<<<<<< HEAD
       BASEIMAGE: "baseimagename" #replace
       MODNAME: "modname" #replace
+=======
+      BASEIMAGE: "lidarr" #replace
+      MODNAME: "flac2mp3" #replace
+>>>>>>> 38e21e1... Initial release
```
Comment (aptalca) [inline .github/workflows/BuildImage.yml:15]: These artifacts need to be removed

Comment (TheCaptain989) [reply .github/workflows/BuildImage.yml]: Gah!  Sorry about that, very sloppy of me.  Now fixed.

**Review by aptalca — APPROVED** [review]

**Pull request merged**